integration-tests/utils/integration-utils.js

```diff
@@ -5,6 +5,8 @@ const { Storage } = require('@google-cloud/storage')
 const { screenshotBucket, screenshotDirPath } = require('../utils/integration-config')
 
 
+const waitForXpathWithOptionVisibleTrue = opts => _.defaults({ visible: true }, opts)
```

Comment:
1. Because Lodash-FP supports auto-currying, `opts => _.defaults({ visible: true }, opts)` can just be `_.defaults({ visible: true })` without needing to change the call site.
2. It'd be good to make it clear that we're setting `visible` to `true` only if a caller doesn't specify a value for `visible`.

```suggestion
const waitForXpathWithVisibleTrueIfUnspecified = _.defaults({ visible: true })
```